shopfloor/models/stock_move_line.py

```diff
@@ -137,3 +137,63 @@ def _check_qty_to_be_done(self, qty_done, split_partial=True, **split_default_va
             self.with_context(bypass_reservation_update=True).product_uom_qty = qty_done
             return (new_line, "lesser")
         return (new_line, "full")
+
+    def replace_package(self, new_package):
+        """Replace a package on assigned move lines"""
```

Comment:
```suggestion
        """Replace a package on an assigned move line"""
```
+ worth adding more details in the docstring